File: JavaScript/8js 1/buildString.js
/*
|--------------------------------------------------------------------------
| String Templates - Bug Fixing #5
|--------------------------------------------------------------------------
|
*/

function buildString(...template) {
   return `I like ${template.join(", ")}!`;
}


const buildString = (...template) => `I like ${template.join(", ")}!`;


function buildString(...template) {
   return "I like " + template.join(", ") + "!";
}


console.log(buildString("Cheese", "Milk", "Chocolate"), "I like Cheese, Milk, Chocolate!");
console.log(buildString("Cheese", "Milk"), "I like Cheese, Milk!");
console.log(buildString("Chocolate"), "I like Chocolate!");

File: Python/8py 1/build_string.py
"""
|--------------------------------------------------------------------------
| String Templates - Bug Fixing #5
|--------------------------------------------------------------------------
|
"""

def build_string(*args):
   return f"I like {', '.join(args)}!"


def build_string(*args):
   return "I like %s!" % ", ".join(args)


def build_string(*args):
   return "I like {}!".format(", ".join(args))


print(build_string('Cheese','Milk','Chocolate'), 'I like Cheese, Milk, Chocolate!')
print(build_string('Cheese','Milk'), 'I like Cheese, Milk!')
print(build_string('Chocolate'), 'I like Chocolate!')